### ISSUE TITLE
Activate or Not: Learning Customized Activation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#117]
# TL;DR
They treated ReLU and Swish in a unified manner and proposed the activation function ACtivationOrNot (ACON) as its general form. The activation function is composed of multiple learnable parameters, and they can freely change whether it is activated or not. They confirmed that this method improves the accuracy of object detection and image retrieval.

![image](https://user-images.githubusercontent.com/24524018/93548644-964f0280-f9a2-11ea-939d-ae48f319e4f3.png)

![image](https://user-images.githubusercontent.com/24524018/93548647-98b15c80-f9a2-11ea-9a76-45848e4f6a64.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.04759

# Submission Dates(yyyy/mm/dd)
2019/09/10

# Authors and institutions
Ningning Ma, Xiangyu Zhang, Jian Sun
- Hong Kong University of Science and Technology
- MEGVII Technology

# Methods

# Results

# Comments
